lib/modules/apostrophe-users/index.js

```diff
@@ -185,6 +185,9 @@ module.exports = {
     options.removeFilters = (options.defaultRemoveFilters || [ 'published' ])
       .concat(options.removeFilters || []);
 
+    options.removeColumns = (options.defaultRemoveColumns || [ 'published' ])
```

Comment:
I'm torn on whether `options.defaultRemoveColumns || ` is needed here as well. I guess it makes sense give the others here and that users might more reasonably be used as content pieces.